From da4d27cc3060098ffe1b79a6f28c686acec53f66 Mon Sep 17 00:00:00 2001
From: Jianing Ren <jianing.ren@rock-chips.com>
Date: Tue, 5 Nov 2019 09:46:15 +0800
Subject: [PATCH] Revert "HACK: arm64: dts: rk3399-android: fixing the adb
 function cannot be used"

This reverts commit 2e4b6184c71d7e05f2550db3501119de554deb96.

Change-Id: I210e3e90855373219c805bec6bca1b1d354318d6
Signed-off-by: Jianing Ren <jianing.ren@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-android.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-android.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-android.dtsi
index e79a2712d8b4..1d3f7722ffa2 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-android.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-android.dtsi
@@ -354,7 +354,7 @@
 };
 
 &usbdrd_dwc3_0 {
-	dr_mode = "peripheral";
+	dr_mode = "otg";
 };
 
 &iep {
-- 
2.35.3

